src/api/chat.ts

```diff
@@ -21,7 +22,7 @@ import {
 export const getItemChat = async (
   id: UUID,
   { API_HOST, axios }: PartialQueryConfigForApi,
-): Promise<ChatMessage[]> =>
+): Promise<ItemChat> =>
```

Comment:
This change is not correct. We return the list of messages only. So unless you update the `ItemChat` type, we should keep using the `ChatMessage[]` no ?